doc-surrealdb_versioned_docs/version-latest/surrealql/statements/define/access/record.mdx

```diff
@@ -170,9 +170,7 @@ ALGORITHM HS512 KEY "secret";
 
 ### With `AUTHENTICATE` clause
 
-The authenticate clause can be used to change the record identifier returned by the `SIGNIN` and `SIGNUP` clauses or replace the identifier provided in the token when authenticating `WITH JWT`. This clause can also be used to log or stop authentication attempts from record users, as it is always executed across signin, signup and token authentication. The clause always expects a record identifier to be returned and will otherwise fail.
-
-Unlike the [`PERMISSIONS`](/docs/surrealdb/surrealql/statements/define/table#defining-permissions) clause, the `AUTHENTICATE` clause is executed only at the time of authentication, resulting in increased performance for queries that only need to be validated at that point. On the other hand, permissions queries are executed in every query and for each record, ensuring that any authorization conditions are verified at the time of the query. The `AUTHENTICATE` clause is a good fit for validating specific conditions that are not expected to change during the lifetime of the session such as the presence of any required token claims.
+In the context of [`DEFINE ACCESS ... TYPE RECORD`](/docs/surrealdb/surrealql/statements/define/access/record), the `AUTHENTICATE` clause is always executed across signin, signup and token authentication. This clause can be used to log or stop authentication attempts from record users, as it is always executed across signin, signup and token authentication. The clause always expects a record identifier to be returned and will otherwise fail.
```

Comment:
The "is always executed across signin, signup and token authentication" sentence is written twice. 

Also, I would not remove the previous paragraphs, as it forces the user to look in two places to get the complete information. The `PERMISSIONS` clause is only relevant for record access, so moving that paragraph to `DEFINE ACCESS` is also not very relevant as it does not impact the other access methods that exist today.